python/ray/data/tests/test_dynamic_block_split.py

```diff
@@ -369,6 +379,91 @@ def foo(batch):
     assert ds.count() == num_blocks_per_task
 
 
+def _test_write_large_data(
```

Comment:
Rather than writing several distinct unit tests, would it make sense to parametrize this function?

Reply:
I think it's pretty much the same thing; is it okay if I keep it like this?

Reply:
IMO explicitly parametrized tests with `@pytest.mark.parametrize` are easier to read, but I won't block on it.